File: BAEKJOON/Python/1000/1200/1271.py
"""
Title : 엄청난 부자2
Link : https://www.acmicpc.net/problem/1271
"""

import sys
input = sys.stdin.readline


if __name__ == "__main__":
    N, M = map(int, input().split())
    print(f'{N // M}\n{N % M}')

File: BAEKJOON/Node/1000/1200/1271.js
const fs = require('fs')
const input = fs.readFileSync("/dev/stdin").toString().trim().split(" ").map(BigInt)
// const input = '1000 100'.split(' ')

let answer = input[0] / input[1] + '\n'
answer += input[0] % input[1]
console.log(answer)
